wd_core.au3

```diff
@@ -1588,15 +1608,30 @@ Func __WD_Error($sWhere, $i_WD_ERROR, $sMessage = Default)
 				$sMsg = $sMsg & ": " & $sMessage
 			EndIf
 
+			$sMsg &= (($iExt) ? (' : Extended = ' & $iExt) : (''))
+
 			__WD_ConsoleWrite($sMsg)
 
 			If @Compiled Then
-				If $_WD_ERROR_MSGBOX And $i_WD_ERROR <> $_WD_ERROR_Success And $i_WD_ERROR < 6 Then MsgBox(16, "WD_Core.au3 Error:", $sMsg)
-				DllCall("kernel32.dll", "none", "OutputDebugString", "str", $sMsg)
+				If $i_WD_ERROR <> $_WD_ERROR_Success Then
+					If $_WD_ERROR_MSGBOX And $i_WD_ERROR < 6 Then
+						Local $iAnswer = MsgBox($MB_ICONERROR + $MB_OKCANCEL, "WD_Core.au3 Error:", $sMsg)
+						If $iAnswer = $IDCANCEL Then
+							$iErr = $_WD_ERROR_UserAbort
+						EndIf
+					EndIf
+					If $_WD_ERROR_OUTPUTDEBUG Then
+						DllCall("kernel32.dll", "none", "OutputDebugString", "str", $sMsg)
+					EndIf
+				EndIf
+			EndIf
+
+			If $_WD_DEBUG = $_WD_DEBUG_Info And $iErr = $_WD_ERROR_UserAbort Then
+				__WD_ConsoleWrite($sFuncName & ': UserAbort on: ' & $sMsg)

```

Comment:
This should go inside the prior If..Endif unless you are wanting this to handle when the incoming error is $_WD_ERROR_UserAbort

Reply:
please take a look [moved: If $_WD_DEBUG = $_WD_DEBUG_Info Then](https://github.com/Danp2/au3WebDriver/pull/244/commits/b312dedc8b39a4e79e20050aaae7fa1315d7f7fa)